packages/runtime-common/package.json

```diff
@@ -15,6 +15,7 @@
     "@babel/types": "^7.20.0",
     "@cardstack/boxel-ui": "workspace:*",
     "@cardstack/ember-template-imports": "workspace:*",
+    "@lukeed/uuid": "^2.0.1",
```

Comment:
Should we use the same UUID library everywhere? This and other packages also have `uuid@9`

Reply:
maybe? this is specifically a uuid lib that is isomorphic and uses Math.random. it's not appropriate to use that for security purposes (however this lib does have a secure v4 uuid generator too that does rely upon a crypto module in the ambient runtime). unsure if our we have any uuid needs are security related in nature.